NetKAN/AviationLights.netkan

```diff
@@ -2,25 +2,17 @@
     "spec_version": "v1.4",
     "identifier":   "AviationLights",
     "name":         "Aviation Lights",
-    "author":       [ "MOARdV", "BigNose", "RPGprayer" ],
-    "$kref":        "#/ckan/github/MOARdV/AviationLights",
+    "author":       [ "LisiasT", "MOARdV", "BigNose", "RPGprayer" ],
+    "$kref":        "#/ckan/spacedock/3208",
     "$vref":        "#/ckan/ksp-avc",
+    "x_netkan_force_v": true,
     "x_netkan_epoch": 1,
     "license":      "CC-BY-NC-SA-4.0",
     "resources": {
-        "homepage": "https://forum.kerbalspaceprogram.com/index.php?/topic/173305-*"
+        "homepage": "https://forum.kerbalspaceprogram.com/index.php?/topic/211781-*"
     },
```

Comment:
```suggestion
```

You don't need this, as it will be pulled from SpaceDock automatically

Reply:
fixed on commit https://github.com/KSP-CKAN/NetKAN/pull/9550/commits/7c84a0deca8c30f939ce14ce993b2ad1da97179f